file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -0,0 +1,70 @@
+INTERFACE zif_aff_ntty_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! $values { @link zif_aff_ntty_v1.data:co_profile }
+    "! $default { @link zif_aff_ntty_v1.data:co_profile.text }
+    ty_profile TYPE c LENGTH 20,
+
+    "! $minimum 0
+    "! $maximum 2147483647
```

Comment:
As type is of type I, you don't need this annotation for maximum. Our schema generator adds the maximum automatically to the JSON Schema. It is only needed if extrema differ from the value range of the built in ABAP type
```suggestion
```